app/vmctl: fix migration process if tenant have no data (#4799)
VictoriaMetrics/VictoriaMetrics: docs/CHANGELOG.md

* FEATURE: [vmbackup](https://docs.victoriametrics.com/vmbackup.html): add support for server-side copy of existing backups. See [these docs](https://docs.victoriametrics.com/vmbackup.html#server-side-copy-of-the-existing-backup) for details.

> * BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl.html): continue the migration process when on of the tenant in the cluster has no metrics. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4796).

Reviewer: ```suggestion
* BUGFIX: [vmctl](https://docs.victoriametrics.com/vmctl.html): don't interrupt the migration process if no metrics were found for a specific tenant. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4796).
```

Author: fixed
